agent_issuance/src/offer/aggregate.rs

```diff
@@ -408,6 +425,7 @@ pub mod tests {
                 OfferEvent::FormUrlEncodedCredentialOfferCreated {
                     offer_id: Default::default(),
                     form_url_encoded_credential_offer,
+                    status: Status::Pending,
```

Comment:
Shouldn't the `Status::Created` be tested somewhere as well?

Reply:
Added it :)